src/main/java/com/google/gtravel/Trip.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+public class Trip {
+  public String userID;
+  public String tripID;
+  public String tripName;
+  public long timestamp;
+
+  public Trip(String _userID, String _tripID, String _tripName, long _timestamp) {
```

Comment:
Remove _prefix and use "this" to differentiate between variables :). E.g. this.userID = userID;
We do not use _ to differentiate between local variables and private variables in java.

Reply:
Thank you, I fixed.